### PR TITLE
Fixes 249: Updated evaluation folder

### LINKED ISSUE #249: Update model_evaluation folder
- [x] Change folder name to "evaluation" 
- [x] Add/modify readme to (vaguely) include info about model and forcings and variables

### PR DESCRIPTION
**Pull request by alaws-USGS**
Closes #249

### CODE REVIEW

**Comment by amsnyder** [comment]
@alaws-USGS - Do you know if we use this filepath in any notebooks or README's that we also need to update?

It would be nice to have a little script that runs through all the content on Github and does a string replacement (with supervision to say yes, replace). Do you or @gzt5142 have something that does this already? I could try to write it if you don't.

**Comment by alaws-USGS** [comment]
@amsnyder I wrote something to do this recently and could adapt it to this. Are you looking for the following on this PR:
1. Find instances of model_evaluation
2. Replace model_evaluation with evaluation
3. Resubmit for review

**Comment by amsnyder** [comment]
> @amsnyder I wrote something to do this recently and could adapt it to this. Are you looking for the following on this PR:
> 
> 1. Find instances of model_evaluation
> 2. Replace model_evaluation with evaluation
> 3. Resubmit for review

Yes! Just in case anything was pulled into a notebook like `model_evaluation/Metrics_StdSuite_v1.ipynb` - we would need to update the filepath. 

I was suggesting some kind of manual review of the changes in case it catches anything we actually didn't want to change - but perhaps we just look at the diff before requesting review on the PR.

**Comment by amsnyder** [comment]
Could you also add that script to this repo? maybe in `doc` and I can just use what you've already pulled together to look for the CONUS404 catalog replacements? I imagine others might also need to use a script like this from time to time

**Comment by alaws-USGS** [comment]
@gzt5142 I hadn't dealt with grep before so this is all a great learning experience for me! @amsnyder I didn't read the supervised part of your comment. I know you can also do this through the VSCode search bar, which is likely the easiest way to do it supervised. Otherwise, its identify notebooks through either CLI (@gzt5142) or script (@alaws-USGS) and do it manually or automatically.

**Comment by alaws-USGS** [comment]
@amsnyder I updated the filepaths with VSCode. Should be good!

**Comment by amsnyder** [comment]
@gzt5142 - would you open a new discussion for "bulk string replacement" and drop your tips in there? I'd like to keep that info around for reference!

**Comment by amsnyder** [comment]
@alaws-USGS - we lost a `/`
image

**Comment by alaws-USGS** [comment]
@amsnyder fixed!

File: evaluation/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+The evaluation folder contains demos, tutorials, and resources for evaluating
+model and forcing datasets. These datasets are not so different from one
+another, as model outputs can be inputs for other models. An example is
+CONUS404, a model output, contains forcings data variables that are used
+in additional models.
+
+An example can be found in the CONUS404 DRB tutorials, which go over the
+process for evaluating forcings inputs (accumulated precipitation, net
+radiation, and mean temperature) of a model ouput (CONUS404).
```
Comment (amsnyder) [inline evaluation/README.md:11]:
```suggestion
radiation, and mean temperature from the CONUS404 dataset), which is actually the output from an application of the WRF model.
```

File: evaluation/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+The *Metrics* notebooks above provide an overview of the different stats
```
Comment (amsnyder) [inline evaluation/README.md:13]:
```suggestion
The *Metrics* notebooks above provides an overview of the different statistics
```

Comment (amsnyder) [inline evaluation/README.md:13]:
```suggestion
The *Metrics* notebooks above provides an overview of the different statistics
```

File: evaluation/tutorials/nwm_streamflow/02_Analysis_StdSuite.ipynb [hunk 3]
```diff
@@ -223,9 +223,9 @@
     "3) <span style=\"color:green; font-size:large\">A set of statistical metrics with which to benchmark.</span>\n",
     "\n",
     "The code to calculate the various NWM metrics has been standardized in\n",
-    "{doc}`/model_evaluation/Metrics_StdSuite_v1`\n",
+    "{doc}`evaluation/Metrics_StdSuite_v1`\n",
     "with usage examples in \n",
-    "{doc}`/model_evaluation/tutorials/stats/Usage_StdSuite_v1`. \n",
+    "{doc}`evaluation/tutorials/stats/Usage_StdSuite_v1`. \n",
```
Comment (amsnyder) [inline evaluation/tutorials/nwm_streamflow/02_Analysis_StdSuite.ipynb:228]:
```suggestion
    "{doc}`/evaluation/tutorials/stats/Usage_StdSuite_v1`. \n",
```

File: evaluation/tutorials/stats/Usage_StdSuite_v1.ipynb [hunk 4]
```diff
@@ -7,7 +7,7 @@
     "# Standard Suite (v1) Metrics -- Usage Examples\n",
     "\n",
     "This notebook will demonstrate how to call the specific functions defined in the \n",
-    "{doc}`/model_evaluation/Metrics_StdSuite_v1`\n",
+    "{doc}`evaluation/Metrics_StdSuite_v1`\n",
```
Comment (amsnyder) [inline evaluation/tutorials/stats/Usage_StdSuite_v1.ipynb:10]:
```suggestion
    "{doc}`/evaluation/Metrics_StdSuite_v1`\n",
```

File: evaluation/tutorials/stats/Usage_StdSuite_v1.ipynb [hunk 5]
```diff
@@ -141,7 +141,7 @@
    "source": [
     "## Import Metric Functions\n",
     "The functions are defined in an \n",
-    "{doc}`/model_evaluation/Metrics_StdSuite_v1`.  \n",
+    "{doc}`evaluation/Metrics_StdSuite_v1`.  \n",
```
Comment (amsnyder) [inline evaluation/tutorials/stats/Usage_StdSuite_v1.ipynb:144]:
```suggestion
    "{doc}`/evaluation/Metrics_StdSuite_v1`.  \n",
```

**Review by amsnyder — COMMENTED** [review]
still some funny business with the slashes

**Comment by alaws-USGS** [comment]
@amsnyder good catches, added suggested changes

**Review by amsnyder — APPROVED** [review]

**Comment by gzt5142** [comment]
> @gzt5142 - would you open a new discussion for "bulk string replacement" and drop your tips in there? I'd like to keep that info around for reference!

I see you already started one -- so I put a few more comments in that thread for the good of the order.

**Pull request merged**